src/blocks/stars.css

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__sizing {
+  width: 16px;
+  height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__sizing:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star {
+  width: 13px;
+  height: 12px;
```

Comment:
Be cautious with fixing the size of elements as it can lead to issues with responsiveness. If there's no strict requirement to fix the size, consider using padding and background-size to control the size of the stars.